pkg/platform/packet/packet_test.go

```diff
@@ -355,3 +356,33 @@ func cmpSliceString(a, b []string) bool {
 
 	return true
 }
+
+func TestAuthTokenIfSet(t *testing.T) {
+	envAuthToken := os.Getenv("PACKET_AUTH_TOKEN")
+	c := config{
+		AuthToken: "Lokomotive",
+	}
+
+	if c.AuthToken == "" {
+		c.AuthToken = os.Getenv("PACKET_AUTH_TOKEN")
+	}
+
+	if c.AuthToken == envAuthToken {
+		t.Errorf("auth_token should not match env auth_token, expected %v, got %v:", c.AuthToken, envAuthToken)
+	}
+}
```

Comment:
This test doesn't make any sense if we do not call `Initialize()`.

Reply:
Is it good, if we call `Initialize` in test? This test tries to duplicate the same behaviour as the ode does.

Reply:
Hmm, I think it should be fine to call `Initialize()` in tests, though we currently don't do it, so it might be complicated to set up. I think we can skip tests on this for now, until we have an easy way to test this.

Reply:
Cool